src/Data/Generic/Rep.purs

```diff
@@ -34,3 +37,6 @@ newtype Argument a = Argument a
 class Generic a rep | a -> rep where
   to :: rep -> a
   from :: a -> rep
+
+toRep :: forall from to. Generic from to => Proxy from -> Proxy to
```

Comment:
Could we reuse `a` and `rep` variables, like in the class declaration?

Reply:
Fixed.